mongo/database.go

```diff
@@ -95,7 +95,7 @@ func (db *Database) Name() string {
 	return db.name
 }
 
-// Collection gets a handle for a collection with the given name configured with the given CollectionOptions.
+// Collection gets a handle for a Collection with the given name configured with the given CollectionOptions.
```

Comment:
I think this one should be lower-case, right?  handle for a collection on the DB?

Reply:
done.